README.md

```diff
@@ -23,6 +23,14 @@ It was created as an answer to deeply recursive Pyanodon recipes, which the tool
 
 Among other things YAFC has Never Enough Items, which is FNEI on steroids. In addition to showing the recipes, it shows which ones you probably want to use, and how much.
 
+## Getting Started
+1. Go to the [releases page](https://github.com/have-fun-was-taken/yafc-ce/releases)
+2. Download the zip file for your given OSX
```

Comment:
"OS", not "OSX", because the sentence is about "given Operating System" if I get it correctly.